README.md

```diff
@@ -1,4 +1,4 @@
-<h2 align="center">python-kasa</h2>
+# python-kasa
```

Comment:
I'm not sure if we want to change this, but I'm not completely against it either.

Reply:
Without this the TOC was rendering the sub headings of the readme at the top level which was messing everything up.  Something to do with the `myst` parser but doing this also prevents sphinx warnings about readme not starting at the right heading level.

Reply:
Okay, it doesn't look that nice with this, but we can revisit that when we clean up the README.md :-)